<!DOCTYPE html>
<html lang="en">
<head>
<meta charset="utf-8">
<title>Above Multi-purpose Free Bootstrap Responsive Template</title>
<meta name="viewport" content="width=device-width, initial-scale=1.0" />
<meta name="description" content="" />
<meta name="author" content="http://webthemez.com" />
<!-- css -->
<link href="css/bootstrap.min.css" rel="stylesheet" />
<link href="css/fancybox/jquery.fancybox.css" rel="stylesheet">
<link href="css/jcarousel.css" rel="stylesheet" />
<link href="css/flexslider.css" rel="stylesheet" />
<link href="css/style.css" rel="stylesheet" />
 
<!-- HTML5 shim, for IE6-8 support of HTML5 elements -->
<!--[if lt IE 9]>
      <script src="http://html5shim.googlecode.com/svn/trunk/html5.js"></script>
    <![endif]-->


</head>
<body>
<div id="wrapper">

	<!-- start header -->
		<header>
        <div class="navbar navbar-default navbar-static-top">
            <div class="container">
                <div class="navbar-header">
                    <button type="button" class="navbar-toggle" data-toggle="collapse" data-target=".navbar-collapse">
                        <span class="icon-bar"></span>
                        <span class="icon-bar"></span>
                        <span class="icon-bar"></span>
                    </button>
                  <a class="navbar-brand" href="index.html"><img src="img/logo.png" alt="logo"/></a>
                </div>
                <div class="navbar-collapse collapse ">
                    <ul class="nav navbar-nav">
                        <li><a href="index.html">Home</a></li> 
						<li><a href="courses.html">Publications</a></li>
						<li class="active"><a href="research.html">Research</a></li>
                        <li><a href="portfolio.html">CV</a></li>
                        <li><a href="Field.html">Field & Collections</a></li>
                    </ul>
                </div>
            </div>
        </div>
	</header><!-- end header -->
	<section id="inner-headline">
	<div class="container">
		<div class="row">
			<div class="col-lg-12">
				<h1 class="pageTitle">Research</h1>
			</div>
		</div>
	</div>
	</section>
	<section id="content">
	<div class="container">

						<div class="row"> 
							<div class="col-md-12">
								<div class="about-logo">
									<p align="justify"	style="font-size:18px;" >What drives speciation? How do neutral and selective processes shape species differentiation? How do physiographic and climatic oscillations shape biogeographic patterns? What are the genomic adaptations behind species’ life history? These questions are the core of my research program, and represent fundamental principles that govern the diversity of life on earth over space and time. To answer these questions I apply a multidisciplinary framework bridging phylogenomics, computational biology, landscape ecology, population genomics, and historical biogeography. My research focuses on three general areas 1) population genetics, by testing the major factors impacting population differentiation over historical timescales, 2) phylogenomics, by studying how genomic architecture impact speciation and levels of polymorphisms across the genome, and 3) comparative genomics, by exploring the adaptations that allow species to have particular life histories. My overriding goal is to unearth the mechanisms driving biodiversity evolution by using genomics and advanced computational biology. Bellow you will find some of the projects that integrate my research program. </p>
									<p></p>
								</div>
							</div>
						</div>  
						<div class="about">

						<div class="row"> 
							<div class="col-md-12">
								<div class="about-logo">
									<h2>Toxin resistance and chemical co-evolution between birds and parasitized army ants of Amazonia </h2>
									<style>
    h2, p {
       margin: 0;
    }
</style>
									<p></p>
								</div>
							</div>
						</div>  
						<br>
							
						 <div class="row">
						 	<div class="col-md-12">
								<p align="justify"	style="font-size:18px;" >Species interactions play a central role in the formation of biodiversity with major diversification events being directly linked to the appearance of novel species associations. After closely studying Amazonian birds in the field for many years, I developed a deep interest in understanding the range of adaptations that allow species to interact. In a recently started project, I am exploring the evolutionary underpinnings that allow a community of birds to parasitize army ant swarms in Amazonia. Army ants forage by forming sprawling swarms that blanket the rainforest floor and prey upon everything in their path. The birds that follow army ants take advantage of the situation and eat invertebrates and small vertebrates trying to escape or being preyed upon by the ants. However, foraging near army ant swarms comes at great risk to the birds. Army ants are highly aggressive and carry a coquetel of toxic compounds that produce a wide variety of destructive physiological effects in vertebrates. This study is exploring the adaptations that enabled these birds to overcome extreme costs to exploit toxic and aggressive ants. I hypothesize that obligatory army ant-following birds evolved a set of adaptations to parasitize army ants colonies by developing mechanisms to neutralize toxins produced by the ants, and sequestering key chemicals from the involuntary consumption of ants to reduce ant’s aggressiveness. I am currently 1) sequencing genomes and transcriptomes of army ant-following birds and testing whether genes that are under positive selection or up/down-regulated in these species are associated with toxin resistance and chemical sequestration, 2) experimentally testing if birds are resistant to ant’s toxins, and 3) testing if the birds are sequestering chemicals derived from ant consumption. In a genomic analysis, I identified genes associated with resistance to toxins produced by the ants and gene ontology terms associated with immune response. Morphological and behavior observations also corroborate our hypothesis (Figure 1). For example, after extensive sampling of specimens across the Amazonia I found in anatomical observations that individuals of obligatory army ant follower species keep a fully developed bursa (a bird organ associated with B cells development in young birds, that atrophies after about six months) across adulthood, which is unique among birds and might be an adaptation to deal with toxin exposure. Recently my colleagues and I obtained an Explorer 21 award to collect samples in the field and execute analyses. This study has the potential to change our understanding of how bird species interact with the environment and exploit resources through highly optimized biological associations. </p>
							</div>
							<div class="row">
								<img src="img/1_Figure_7.jpg" alt=""	width="800" >
								<div class="space"></div>
							</div>
							
						</div>	

						<div class="row"> 
							<div class="col-md-12">
								<div class="about-logo">
									<h2>Genomic architecture controls biogeographic structuring in Amazonian birds </h2>
									<p></p>
								</div>
							</div>
						</div>  
						<br>
							
						 <div class="row">
						 	<div class="col-md-12">
								<p align="justify"	style="font-size:18px;" >Multiple intrinsic and extrinsic processes affect genetic diversity across organisms' genomes. Mutation, recombination, genetic drift, and migration operate simultaneously and heterogeneously and can directly interfere with our ability to estimate the phylogenetic relationships among species.  The fast development of new DNA sequencing technologies is opening new horizons on the use of whole-genome sequencing (WGS) on non-model organisms, allowing us to explore in more detail how microevolutionary processes shape phylogenetic signal across the genome. In my current position, as a postdoctoral fellow, I generated nearly 100 whole genomes for three species restricted to the eastern Amazon, to explore how intrinsic (e.g., mutation and recombination) and extrinsic (e.g., selection and gene flow) genomic processes can affect the phylogenetic signal and levels of polymorphism. Here we are leveraging the power of supervised machine learning (SML) to show that phylogenetic signals and demographic parameters vary considerably across the genome but in predictable ways. We found that approximately 30% of the genome of studied species is affected by the interplay between meiotic recombination and natural selection, which violated models that assume neutral evolution and subsequently bias key parameter estimations </p>
							</div>
							<div class="row">
								<img src="img/1_Figure_6.jpg" alt=""	width="800" >
								<div class="space"></div>
							</div>
							
						</div>	


					
					<div class="about">
					
						<div class="row"> 
							<div class="col-md-12">
								<div class="about-logo">
									<h2>Microevolutionary dynamics show tropical valleys are deeper for montane birds of the Atlantic Forest. </h2>
									<p></p>
								</div>
							</div>
						</div>  
						<br>
							
						 <div class="row">
							<div class="col-md-6">
								<img src="img/1_Figure_1.jpg" alt="">
								<div class="space"></div>
							</div>
							<div class="col-md-6">
								<p align="justify"	style="font-size:18px;" >In 1967, Dan Janzen proposed a profound mechanism that could explain the high species richness in tropical mountains, known as the “mountain passes are higher in the tropics” hypothesis. Since its description, a large number of studies have been corroborating assumptions associated with this process. However, the vast majority of studies focused on comparisons of highly divergent lineages, occurring in contrasting latitudes, observing the effects of climate variability in space over deep evolutionary time frames. Using genome-wide markers from populations of 21 species, climatic modeling, and statistical modeling we were able to demonstrate that species’ tropical populations are under greater ecological constraints and thus have undergone greater genomic differentiation than their populations in subtropical areas. These findings support the main assumptions of Janzen’s hypothesis and show the phenomenon is observable at a very early stage of the speciation process and small spatial scale. By focusing on the microevolutionary scale, we were able to exclude the noise generated by time, which leads populations to diverge in their traits and history and infer that adaptation to local climate might be one of the first processes to operate in driving the contrasting diversity patterns across latitude. Our results provide clear evidence for how a narrow latitudinal gradient can shape microevolutionary processes and contribute to broader scale biodiversity patterns. </p>
							</div>
						</div>	



					<div class="about">
					
						<div class="row"> 
							<div class="col-md-12">
								<div class="about-logo">
									<h2>Climatic dynamics and topography control genetic variation in Atlantic Forest montane birds. </h2>
									<p></p>
								</div>
							</div>
						</div>  
						<br>
							
						 <div class="row">
							<div class="col-md-6">
								<img src="img/1_Figure_2.jpg" alt="">
								<div class="space"></div>
							</div>
							<div class="col-md-6">
								<p align="justify"	style="font-size:18px;" >Montane organisms responded to Quaternary climate change by tracking suitable habitat along elevational gradients. However, it is unclear whether these past climatic dynamics generated predictable patterns of genetic diversity in co-occurring montane taxa. We integrated paleoclimatic data and a model selection approach based on a machine learning algorithm for testing the demographic history of five co-distributed bird species occurring in the southern Atlantic Forest sky islands. Our results suggest that the heterogeneous distribution of genetic variation in birds of the Atlantic Forest sky islands is associated with the interplay between topography and climate of distinct mountains, leading to specific patterns of genetic diversity that can be used to predict future responses to climate change. </p>
							</div>
						</div>	


					<div class="about">
					
						<div class="row"> 
							<div class="col-md-12">
								<div class="about-logo">
									<h2>Quaternary climate changes as speciation drivers in the Amazon floodplains. </h2>
									<p></p>
								</div>
							</div>
						</div>  
						<br>
							
						 <div class="row">
							<div class="col-md-6">
								<img src="img/1_Figure_3.jpg" alt="">
								<div class="space"></div>
							</div>
							<div class="col-md-6">
								<p align="justify"	style="font-size:18px;" >The role of climate as a speciation driver in the Amazon has long been discussed. Phylogeographic approaches did not recover consistent demographic oscillations, not supporting conclusive evidence on the role of climatic changes in promoting diversification in the Amazon. We conducted a comparative phylogenomic analysis of floodplain forest birds to explore the effects of historical environmental changes and current connectivity on population differentiation. Our data suggest that changes in the fluvial landscape induced by climate variation during the Mid- and Late Pleistocene drove population isolation, leading to diversification with subsequent secondary contact. These results provided one of the most substantial pieces of evidence to date supporting that climatic oscillations are intimately linked to genetic differentiation in the Amazon. </p>
							</div>
						</div>	



						<div class="about">
					
						<div class="row"> 
							<div class="col-md-12">
								<div class="about-logo">
									<h2>Phenotypic and genetic structure support gene flow generating gene tree discordances in an Amazonian floodplain endemic species. </h2>
									<p></p>
								</div>
							</div>
						</div>  
						<br>
							
						 <div class="row">
							<div class="col-md-6">
								<img src="img/1_Figure_4.jpg" alt="">
								<div class="space"></div>
							</div>
							<div class="col-md-6">
								<p align="justify"	style="font-size:18px;" > In this study, we used an integrative approach, including genomic data, mitochondrial DNA Sanger sequencing, and morphological data to investigate species limits and phylogenetic relationships in the face of gene flow in an Amazonian endemic species (Myrmoborus lugubris: Aves). Before populations become independent evolutionary lineages, the effects of microevolutionary processes tend to generate complex scenarios of diversification that may affect phylogenetic reconstruction. Not accounting for gene flow in species tree estimates can directly impact topology, effective population sizes, and branch lengths, and the resulting estimation errors are still poorly understood in wild populations. Our results demonstrated that failing to account for gene flow when estimating phylogenies at shallow divergence levels can generate topological uncertainty, which can nevertheless be statistically well supported, and that model testing approaches using simulated data can be useful tools to test alternative phylogenetic hypotheses. </p>
							</div>
						</div>	


						<div class="about">
					
						<div class="row"> 
							<div class="col-md-12">
								<div class="about-logo">
									<h2>Recent chapters of neotropical history overlooked in phylogeography: shallow divergence explains phenotype and genotype uncoupling in Antilophia manakins. </h2>
									<p></p>
								</div>
							</div>
						</div>  
						<br>
							
						 <div class="row">
							<div class="col-md-6">
								<img src="img/1_Figure_5.jpg" alt="">
								<div class="space"></div>
							</div>
							<div class="col-md-6">
								<p align="justify"	style="font-size:18px;" > Establishing links between phenotypic and genotypic variation is a central goal of evolutionary biology, as they might provide valuable insights into evolutionary processes shaping genetic and species diversity in nature. One of the more intriguing possibilities is when no genetic divergence is found to be associated with conspicuous phenotypic change. However, a finding of phenotypic distinctiveness with weak or no population genetic structure may simply result from low statistical power to detect shallow genetic divergences when small data sets are used. Here, we used a genomic data set to explore genome-wide divergence between two species of Antilophia manakins, which are phenotypically distinct yet lack strong genetic differentiation. Our results revealed a clear population structure that matches the two phenotypes, supporting the idea that smaller data sets lacked the power to detect recent divergence events. One implication of our study is that genomic data sets are necessary to help redefine local conservation programs by revealing cryptic population structure that may be key to population management. </p>
							</div>
						</div>	



						
										
					</div>
									
				</div>
	</section>
		<footer>
		<div class="container">
			<div class="row">
				<div class="col-lg-3">
					<div class="widget">
						<h5 class="widgetheading">Contact:</h5>
						<address>
						<strong>The American Museum of Natural History</strong><br>
						Central Park West at 79th Street<br>
						 New York, NY 10024-5192</address>
						<p>
						<i class="icon-envelope-alt"></i>Email: <strong>biogrego@yahoo.com.br<strong>
						</p>
					</div>
				</div>
			</div>
		</div>
	<div id="sub-footer">
		<div class="container">
			<div class="row">
				<div class="col-lg-6">
					<div class="copyright">
						<p>
							<span>&copy; Above Site All right reserved. Template By </span><a href="http://webthemez.com" target="_blank">WebThemez</a>
						</p>
					</div>
				</div>
				<div class="col-lg-6">
					<ul class="social-network">
						<li><a href="#" data-placement="top" title="Facebook"><i class="fa fa-facebook"></i></a></li>
						<li><a href="#" data-placement="top" title="Twitter"><i class="fa fa-twitter"></i></a></li>
						<li><a href="#" data-placement="top" title="Linkedin"><i class="fa fa-linkedin"></i></a></li>
						<li><a href="#" data-placement="top" title="Pinterest"><i class="fa fa-pinterest"></i></a></li>
						<li><a href="#" data-placement="top" title="Google plus"><i class="fa fa-google-plus"></i></a></li>
					</ul>
				</div>
			</div>
		</div>
	</div>
	</footer>
</div>
<a href="#" class="scrollup"><i class="fa fa-angle-up active"></i></a>
<!-- javascript
    ================================================== -->
<!-- Placed at the end of the document so the pages load faster -->
<script src="js/jquery.js"></script>
<script src="js/jquery.easing.1.3.js"></script>
<script src="js/bootstrap.min.js"></script>
<script src="js/jquery.fancybox.pack.js"></script>
<script src="js/jquery.fancybox-media.js"></script> 
<script src="js/portfolio/jquery.quicksand.js"></script>
<script src="js/portfolio/setting.js"></script>
<script src="js/jquery.flexslider.js"></script>
<script src="js/animate.js"></script>
<script src="js/custom.js"></script>
</body>
</html>